### PR TITLE
fix apache apisix icon not show issue

### PR DESCRIPTION
**Pull request by lilien1010**
get ico link from index.html ，so that the ico link keep Consistent

### CODE REVIEW

File: src/layout/components/Sidebar/SidebarLogo.vue [hunk 1]
```diff
@@ -69,6 +69,29 @@ export default class extends Vue {
   @Prop({ required: true }) private collapse!: boolean
 
   private title = 'APISIX'
+
+  get getIco() {
```
Comment (juzhiyuan) [inline src/layout/components/Sidebar/SidebarLogo.vue]: We could use an elegant way to fix this issue... Let me check.

**Comment by juzhiyuan** [comment]
After several attempts, I find that the icon only disappears when we use `history` mode, the Dashboard is using `hash` mode by default.

If using `history` mode suits your needs, please update the codes in `SidebarLogo.vue` file: Just use `favicon.ico` instead of `/favicon.ico`

image

File: src/layout/components/Sidebar/SidebarLogo.vue (hunk 1 above)
Comment (lilien1010) [reply src/layout/components/Sidebar/SidebarLogo.vue]: sure thisPR is too complicated, but I think use the same link from link.rel will keep the Icon link consistent

Comment (juzhiyuan) [reply src/layout/components/Sidebar/SidebarLogo.vue]: I'd love to know if the latest version of dashboard is working on your side?

**Pull request closed without merging**